resources/other/git.md

```diff
@@ -1,46 +1,171 @@
-# Git Tutorial
+# <img src="https://git-scm.com/images/logos/downloads/Git-Logo-2Color.png" width="100px" style="vertical-align: bottom"/> Cookbook
+
+
+All software teams use version control to keep track of and share their technical artifacts. Git is the most commonly used versions control system in use today. While Git is incredibly flexible, it can be challenging to learn to use. Below is a task-oriented list of things you might want to accomplish and a short description of how to accomplish each task. There are a variety of ways to accomplish many of these actions, this description focuses on the simplest common mechanism. The `highlighted text` are commands you can type, but you will need to specify text in `<...>` blocks.
+
+# TOC
+
+* Recipies
+
+ 1. [Getting a repo](#clone)
+ 1. [Receiving work](#pull)
+ 1. [Adding a file](#add)
+ 1. [Committing work](#commit)
+ 1. [Sharing work](#push)
+ 1. [Listing existing branches](#list)
+ 1. [Switching to a branch](#switch)
+ 1. [Moving work into a new branch](#move) 
+
+* [Additional content](#additional)
+
+* [Resources](#resources)
+
+<a name="clone"></a>
+## Getting a repo 
+
+Before you can use a repo you need to get a local copy on your machine from the remote server. To do this, you can specify the repo you want checked out.
+
+`git clone git@github.students.cs.ubc.ca:<org>/<repo>.git`
+
+This should work for public repos, but if you are checking out a private repo you may need to configure your username and password.
+
+<a name="pull"></a>
+## Receiving work
+
+If there have been updates on your repo (e.g., your teammates have made commits), you can update your local copy with any changes they have shared with the remote server.
+
+`git pull` 
+
+<a name="add"></a>
+## Adding a file
```

Comment:
Marking files to commit

Might be a bit clearer that you still need to commit the staged files.